admin/filters/class-stale-content-filter.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Registers the filter for filtering stale content.
+ */
+class WPSEO_Stale_Content_Filter extends WPSEO_Abstract_Post_Filter {
+
+	/**
+	 * Returns the query value this filter uses.
+	 *
+	 * @return string The query value this filter uses.
+	 */
+	public function get_query_val() {
+		return 'stale-content';
+	}
+
+	/**
+	 * Modify the query based on the seo_filter variable in $_GET
+	 *
+	 * @param string $where Query variables.
+	 *
+	 * @return string The modified query.
+	 */
+	public function filter_posts( $where ) {
+		if ( ! $this->is_filter_active() ) {
+			return $where;
+		}
+
+		global $wpdb;
+
+		$where .= sprintf( ' AND ' . $wpdb->posts . '.post_modified < "%s" ', $this->date_threshold() );
+		$where .= sprintf(
+			' AND ' . $wpdb->posts . '.ID IN( SELECT post_id FROM ' . $wpdb->postmeta . ' WHERE meta_key = "%s" AND meta_value = "1" ) ',
+			WPSEO_Meta::$meta_prefix . WPSEO_Cornerstone::META_NAME
+		);
+
+		return $where;
+	}
+
+	/**
+	 * Returns the label for this filter.
+	 *
+	 * @return string The label for this filter.
+	 */
+	protected function get_label() {
+		return __( 'Stale content', 'wordpress-seo' );
+	}
+
+	/**
+	 * Returns a text explaining this filter.
+	 *
+	 * @return string The explanation.
```

Comment:
To keep things consistent, I'd suggest having the `@return` description conform to one standard, i.e. `The explanation for this filter.` as per the documentation in the `get_label()` method.